examples/slu/slurp/data_utils/decode_resample.py

```diff
@@ -0,0 +1,114 @@
+# ! /usr/bin/python
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+import json
+import multiprocessing
+import os
+from pathlib import Path
+
+import sox
+import tqdm
+from sox import Transformer
+from tqdm.contrib.concurrent import process_map
+
+FIELD_AUDIO = "audio_filepath"
+FIELD_TEXT = "text"
+FIELD_DATA_DIR = "data_dir"
+
+
+def process(x):
+    if not isinstance(x[FIELD_TEXT], str):
```

Comment:
Document what this function is doing, plus how to use this entire script. Write example call of this script at the top.